CHI_TECH/ChiMesh/Cell/cell_slab.h

```diff
@@ -15,6 +15,12 @@ class chi_mesh::CellSlab : public chi_mesh::Cell
   int     v_indices[2];
   int     edges[2];
   Vector  face_normals[2];
+
+public:
```

Comment:
The member variables should be initialized explicitly in the constructor

Reply:
I've added default values to the variables however, this is just temporary. I want to have a look at the cells in general. I want contributors to be able to populate the cells in a sane way and I have some ideas on how to do this. Do you mind if we revisit this again?